config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  resources :appointments
+  resources :doctors
+  root "users#index"
+  resources :users
```

Comment:
- [ ] I suggest that you only create the routes you will really need. Currently you are creating all possible routes.